### ISSUE TITLE
[Badges] The line spacing of the new badge system in too small for Arabic 

### ISSUE THREAD

**Issue by FrancyCavallari** [vgwb/Antura#314]
![image](https://user-images.githubusercontent.com/17926293/169874913-9afdf377-71f2-4cd8-9add-bb18fbb01afd.png)
![image](https://user-images.githubusercontent.com/17926293/169874914-c154daf5-765a-4d0d-aad5-0de6fd1bfc1e.png)

**Comment by CatmanIta** [comment]
I'd rather not hack this for each language, let me try finding a middle ground

**Comment by FrancyCavallari** [comment]
Yes, let's increase the spacing a bit for all languages/editions
